porcupine/plugins/mergeconflict.py

```diff
@@ -151,6 +152,9 @@ def setup_displayers(tab: tabs.FileTab) -> None:
 
 def on_new_tab(tab: tabs.Tab) -> None:
     if isinstance(tab, tabs.FileTab):
+        [linenums] = [
+            child for child in tab.left_frame.winfo_children() if isinstance(child, LineNumbers)
+        ]
```

Comment:
With a loop instead of this, fold plugin could be made to work when linenumbers plugin is disabled. Also applies to fold plugin.